test/fixtures/with-config/with-config.test.js

```diff
@@ -14,26 +14,18 @@ const hooks = [
 
 describe('with-config', () => {
   buildFixture('with-config', () => {
-    expect(consola.warn).toHaveBeenCalledTimes(5)
+    expect(consola.warn).toHaveBeenCalledTimes(6)
     expect(consola.fatal).toHaveBeenCalledTimes(0)
     expect(consola.warn.mock.calls).toMatchObject([
-      [
-        'Unknown mode: unknown. Falling back to universal'
-      ],
-      [
-        `Invalid plugin mode (server/client/all): 'abc'. Falling back to 'all'`
-      ],
+      ['Unknown mode: unknown. Falling back to universal'],
+      ['Invalid plugin mode (server/client/all): \'abc\'. Falling back to \'all\''],
       [{
-        message: 'Found 2 plugins that match the configuration, suggest to specify extension:',
-        additional: expect.stringContaining('plugins/test.json')
+        'additional': expect.stringContaining('plugins/test.json'),
+        'message': 'Found 2 plugins that match the configuration, suggest to specify extension:'
       }],
-      [
-        'Using styleResources without the nuxt-style-resources-module is not suggested and can lead to severe performance issues.',
-        'Please use https://github.com/nuxt-community/style-resources-module'
-      ],
-      [
-        'Notice: Please do not deploy bundles built with analyze mode, it\'s only for analyzing purpose.'
-      ]
+      ['Please use `build.postcss` in your nuxt.config.js instead of an external config file. Support for such files will be removed in Nuxt 3.'],
```

Comment:
```suggestion
      ['Please use `build.postcss` in your nuxt.config.js instead of an external config file. Support for such files will be removed in Nuxt 3 as they remove all defaults set by Nuxt and can cause severe problems with features like alias resolving inside your CSS.'],
```